README.md

```diff
@@ -57,7 +57,8 @@ Want to contribute to act? Awesome! Check out the [contributing guidelines](CONT
 
 ## Building from source
 
-* Install Go tools 1.11+ - (https://golang.org/doc/install)
+* Install Go tools 1.11.4+ - (https://golang.org/doc/install)
+* Have a working Docker v18.09.1+ client - local/Docker for Mac/Docker for Windows
```

Comment:
This should not be needed; the client can do API version negotiation; see https://github.com/docker/docker.github.io/pull/7710#discussion_r244218735

(thinking of making an opt for that, or see if we can make it the default)

Reply:
whoops, missed your note about that;

> this is probably something that can change, but its true atm

☺️🤗

Reply:
thanks @thaJeztah for the suggestions, i've incorporated that change just now to enable client negotiation on master

Reply:
@SvenDowideit given the change i made to enable client negotiation, can you remove the requirement for having the docker client?